IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TrackFacilityPart/DocPropertySet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_TrackFacilityPart" UniqueId="011923c1-fbf2-84f9-6215-e3ea43caebc9" ApplicableType="IfcFacilityPart" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsBallastTrack_3GWQlZqZbCOhyZObZZzqs" />
+		<DocProperty xsi:nil="true" href="IsCWR_1mrqmXkov0pfRjK0zoCDrB" />
+		<DocProperty xsi:nil="true" href="IsSunExposed_3jUxqj7iX3zerVO_VEXgG7" />
+		<DocProperty xsi:nil="true" href="DetailedDrawingID_0v98BA4e53VwGZS60uKm8C" />
+		<DocProperty xsi:nil="true" href="TrackPartType_3smcaufEP68Po49oZGOONY" />
```

Comment:
This is double typing, the PEnum lists a set of FacilityPart PDTs

Reply:
Removed https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89